data/systemd/snapd.core-fixup.service.in

```diff
@@ -8,7 +8,7 @@ Documentation=man:snap(1)
 
 [Service]
 Type=oneshot
-ExecStart=@libexecdir@/snapd/snapd.core-fixup.sh
+ExecStart=@libexecdir@/snapd.core-fixup.sh
```

Comment:
Hmm, so is libexecdir set so that it contains the `/snapd` suffix already?

Reply:
I don't like this in particular, because I'd rather the automatic `--libexecdir` passed by `%configure` or `dh` Just Work(TM) for this.

Reply:
> Hmm, so is libexecdir set so that it contains the /snapd suffix already?

Right, we're passing `--libexecdir=<whatever>/snapd`, that's why it's a bit confusing. I share @Conan-Kudo sentiment that we should rather use `--libexecdir=<platform-whatever>` and then use this in the Makefiles:
```
libexecsnapddir= $(libexecdir)/snapd
libexecsnapd_PROGRAMS = ...
libexecsnapd_SCRIPTS = ...
```
Just didn't want to make this PR too big, but if we are in agreement I can include those changes here as well.

Reply:
👍

Reply:
Yeah, let's make it obvious, even if it needs to be longer.

Reply:
Updated.